src/lib/components/Carousel.tsx

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, { useRef } from 'react';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import Slider from 'react-slick';
+import { settings, styles } from '../constants/carousel-config';
+
+const Carousel: React.FC = ({ children }) => {
+  const sliderRef = useRef<Slider>(null);
+
+  const prev = () => {
+    sliderRef.current?.slickPrev();
+  };
+
+  const next = () => {
+    sliderRef.current?.slickNext();
+  };
+
+  return (
+    <div style={styles.wrapper}>
+      <Slider {...settings} ref={sliderRef}>
```

Comment:
is there `setting` prop? so that we could just pass the config instead of spreading ?

Reply:
no such thing. [spreading](https://react-slick.neostack.com/docs/get-started) is how react-slick docs suggest doing things